### ISSUE TITLE
Wrong Password

### ISSUE THREAD

**Issue by FrederikOnGitHub** [mikrohash/isf-jclient#1]
It always says I have the wrong Password. I reseted it and it still doesn't work. No Copy-Paste, no typing... Am on a MacBook with 10.13.3 in Terminal.

**Comment by jgordor** [comment]
Same here, ubuntu server (terminal), also recovered from http://iotaspam.com/account?p=recover, no way to login

**Comment by jgordor** [comment]
Also, in the start configuration menu, if you are not fast introducing the info the app crashes. I'm executing the jar in `/var/apps/iota-spamfund/current` and the log dir is created in `/var/apps/iota-spamfund/current/logs` (so seems that is looking at the wrong dir):

```
[16:54:08] [FileMngr]  [ERR] /var/apps/iota-spamfund/logs/2018-02-08_16-49-48.txt (No such file or directory)

java.io.FileNotFoundException: /var/apps/iota-spamfund/logs/2018-02-08_16-49-48.txt (No such file or directory)
	at java.io.FileOutputStream.open0(Native Method)
	at java.io.FileOutputStream.open(FileOutputStream.java:270)
	at java.io.FileOutputStream.<init>(FileOutputStream.java:213)
	at java.io.FileOutputStream.<init>(FileOutputStream.java:162)
	at iota.FileManager.write(FileManager.java:96)
	at iota.ui.UIManager.saveLogs(UIManager.java:65)
	at iota.ui.UIManager.print(UIManager.java:51)
	at iota.ui.UIManager.readLine(UIManager.java:109)
	at iota.ui.UIManager.askQuestion(UIManager.java:120)
	at iota.Configs.askForAccountData(Configs.java:53)
	at iota.SpamFundAPI.keepSendingUntilSuccess(SpamFundAPI.java:134)
	at iota.Configs.askForAccountData(Configs.java:58)
	at iota.Configs.generateConfigs(Configs.java:41)
	at iota.Configs.init(Configs.java:31)
	at iota.Main.main(Main.java:17)
```

**Comment by vuapo-eth** [comment]
both bugs should be fixed in v1.0.2

**Comment by laubblaeser** [comment]
Also had this bug in v1.0.0. It's fixed now.
